esp/bindings/SOAP/Platform/soapbind.cpp

```diff
@@ -286,7 +286,7 @@ static IPropertyTree *createSecClientConfigBuf(const char *clientCertBuf, const
     if (!isEmptyString(caCertsBuf))
     {
         IPropertyTree *ca = ensurePTree(verify, "ca_certificates");
-        ca->setProp("@cacertbuf", caCertsBuf);
+        ca->setProp("@pem", caCertsBuf);
```

Comment:
Since the content is a large buffer, likely with newlines, I would remove the "@" making it an element instead of an attribute.